osu.Framework/Graphics/Containers/GridContainerContent.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Framework.Graphics.Containers
+{
+    /// <summary>
+    /// Implements a jagged array behavior with element change notifications
+    /// </summary>
+    public class GridContainerContent
```

Comment:
Useful internally, only.
```suggestion
    internal class GridContainerContent
```

Reply:
This means that `GridContainer.Content` have to be internal which is not possible since it is used in some game classes.
![image](https://user-images.githubusercontent.com/22450664/86749229-bfb7ed80-c045-11ea-8ec9-924403b1a733.png)

Reply:
`public` class with a `private` or `internal` ctor works (i.e. always use the implicit operator).